packages/drivers/driver-base/src/documentDeltaConnection.ts

```diff
@@ -603,8 +603,17 @@ export class DocumentDeltaConnection
 			// Socket can be disconnected while waiting for Fluid protocol messages
 			// (connect_document_error / connect_document_success), as well as before DeltaManager
 			// had a chance to register its handlers.
-			this.addTrackedListener("disconnect", (reason) => {
-				const err = this.createErrorObject("disconnect", reason);
+			this.addTrackedListener("disconnect", (reason, description) => {
+				const err = this.createErrorObject(
+					"disconnect",
+					`${reason}${
+						typeof description === "object"
```

Comment:
Can you please add reference here to https://socket.io/docs/v4/client-api/#event-disconnect ?
I do not see documentation telling anything about `description` shape.
Also we should avoid using instanceof if possible, as it does not work for objects created in another iframe (unlikely to be here, but in general - you never know where objects might be created).
Can we instead use createErrorObject() with this object?

Here is an example of what we get for websocket errors:
socket.io (connect_error): websocket error: {"description":{"isTrusted":true},"type":"TransportError"}

Reply:
Actually I looked into socket.io code in their repo and seems like they never add this arg. So I am removing this.